### DISCUSSION TITLE
Help I just started c programming I am an absolute noob

### DISCUSSION THREAD

**Discussion by gjon3s** [community/community#21487] [Other Feature Feedback, Questions, & Ideas]
<p>So I wrote this programe but I am getting the wrong output </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/99a9062cb0bac8c54d6cc82aad58282983c32482.jpeg" data-download-href="/uploads/short-url/lVlbv08d1a7EaGqV3sg0P2AVDZE.jpeg?dl=1" title="Screenshot_20200809-143452" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/69417584/181064991-927994ff-7439-4db0-a810-a95e25c31dae.jpeg" alt="Screenshot_20200809-143452" data-base62-sha1="lVlbv08d1a7EaGqV3sg0P2AVDZE" width="690" height="80" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/99a9062cb0bac8c54d6cc82aad58282983c32482_2_690x80.jpeg, https://aws1.discourse-cdn.com/github/original/2X/9/99a9062cb0bac8c54d6cc82aad58282983c32482.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/99a9062cb0bac8c54d6cc82aad58282983c32482.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/99a9062cb0bac8c54d6cc82aad58282983c32482_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20200809-143452</span><span class="informations">799×93 2.51 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> !

**Comment by gjon3s** [comment]
<p>This is the programme </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/baf212cc2aabf06ad5230b0b84c55625b485b6ca.jpeg" data-download-href="/uploads/short-url/qFNrRBVlNFJWupfNjVzuXPwWcFs.jpeg?dl=1" title="Screenshot_20200809-143432" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/69417584/181064992-eb503d61-0ccd-48ac-923c-ecff98a743e3.jpeg" alt="Screenshot_20200809-143432" data-base62-sha1="qFNrRBVlNFJWupfNjVzuXPwWcFs" width="690" height="188" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/baf212cc2aabf06ad5230b0b84c55625b485b6ca_2_690x188.jpeg, https://aws1.discourse-cdn.com/github/original/2X/b/baf212cc2aabf06ad5230b0b84c55625b485b6ca.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/baf212cc2aabf06ad5230b0b84c55625b485b6ca.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/baf212cc2aabf06ad5230b0b84c55625b485b6ca_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20200809-143432</span><span class="informations">800×218 23 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>The problem is that you have a semicolon after <code>for (...)</code>. That way the semicolon (empty instruction) becomes the loop body, and the <code>printf()</code> call is executed <em>after</em> the loop terminates. <code>f = 320</code> is the first value of <code>f</code> for which <code>f &lt;= 300</code> fails.</p>
<p>Try to find an editor/IDE that can do autoindenting, when the <code>printf()</code> call is indented to the same level as the <code>for</code> you’ll know something is wrong. :wink:</p>
<p>And in the future when posting code, please use a markdown code block (enclosed in ```) instead of screenshots, it’s much easier to read that way.</p>

**Comment by gjon3s** [comment]
<p>Thanks, it was a screenshot cuz I couldn’t copy paste from jdoodle.</p>
